local.bib

```diff
@@ -0,0 +1,41 @@
+@inproceedings{2024SPIE13101.59Ftmp,
+author = {Tiago {Ribeiro} and Dave Mills and Andy Clements},
+title = "{Replacing DDS with Kafka as middleware technology for the Rubin Observatory Control System}",
+volume = 13101,
+year = 2024,
+pages = {13101-59, in press},
+editor = {Jorge Ibsen and Gianluca Chiozzi},
+series = "Society of Photo-Optical Instrumentation Engineers (SPIE) Conference Series",
+booktitle = "Software and Cyberinfrastructure for Astronomy VIII",
+url = {https://dx.doi.org/doi#here},
+}
+
+@INPROCEEDINGS{2022SPIE12189E..0MG,
```

Comment:
Usually, listing each and every author for references with many authors is not necessary, and wastes space that could be used to add content. Check the conference style guide for how many authors you need to explicitly list before just putting down "et al.".

Reply:
This is using the conference provided bibstyle which doesn't limit authors. Reading the conference instructions to authors I see no explicit mention of author list limits so I can only assume they are wanting this to happen...